### PR TITLE
Remove Concord project

### PR DESCRIPTION
**Pull request by jaredpar**
At this point the Concord project exists just to fix a signing issue in
the Debugger NuGet packages.

This project is causing problems in our port to the new SDK because it's
a Net 45 portable project that is referenced by Net 20 projects.  The
old build silently ignores this incompatibility but the new SDK takes
issue with it, and the method it uses to pass DLLs around.

Taking the simpler approach here of just fixing the signing issues (side
thread going with debugger team) and removing the project.

### CODE REVIEW

**Comment by jaredpar** [comment]
CC @tmat, @KevinH-MS, @jmarolf for review.

**Comment by jaredpar** [comment]
CC @tannergooding as well.

File: build/Targets/Dependencies.props [hunk 1]
```diff
@@ -27,8 +27,8 @@
     <MicrosoftMetadataVisualizerVersion>1.0.0-beta1-61531-03</MicrosoftMetadataVisualizerVersion>
     <MicrosoftNETCoreRuntimeCoreCLRVersion>1.1.0</MicrosoftNETCoreRuntimeCoreCLRVersion>
     <MicrosoftVisualBasicVersion>10.1.0</MicrosoftVisualBasicVersion>
-    <MicrosoftVisualStudioDebuggerEngineVersion>15.0.26201-alpha</MicrosoftVisualStudioDebuggerEngineVersion>
-    <MicrosoftVisualStudioDebuggerMetadataVersion>15.0.26201-alpha</MicrosoftVisualStudioDebuggerMetadataVersion>
+    <MicrosoftVisualStudioDebuggerEngineVersion>15.0.26201-gamma</MicrosoftVisualStudioDebuggerEngineVersion>
```
Comment (KevinH-MS) [inline build/Targets/Dependencies.props]: aside...which language are we going to switch to when encounter the "omega rollover" bug? :)

**Comment by tmat** [comment]
👍

File: build/Targets/Dependencies.props (hunk 1 above)
Comment (jaredpar) [inline build/Targets/Dependencies.props]: We will officially switch to jared speak

-awesome
-why
-hopefullythisworks

**Pull request merged**